library/src/main/kotlin/me/saket/filesize/FileSize.kt

```diff
@@ -49,7 +49,8 @@ class FileSize(private val bytes: Long) : Comparable<FileSize> {
       is Byte,
       is Short,
       is Int,
-      is Long -> Math.multiplyExact(bytes, other.toLong())
+      is Long,
+      -> Math.multiplyExact(bytes, other.toLong())
```

Comment:
Oof this looks terrible. Considering that this is a tiny project with just two source files, I'm okay with not having an auto-formatter. Can you remove this commit that introduces ktlint + spotless from this PR? I'll merge the rest.

Reply:
c926e7aed3eb8881b0f19fc3539e64bd1fdf8427